pkg/plugins/autodiscovery/githubaction/main.go

```diff
@@ -107,8 +114,10 @@ type Spec struct {
 type GitHubAction struct {
 	// credentials defines the credentials to use to authenticate to the git provider
 	credentials map[string]gitProviderToken
-	// files defines the accepted GitHub Action workflow file name
+	// files defines the accepted Action workflow file name
 	files []string
+	// actions defines the accepted Composite Action name
+	actions []string
```

Comment:
What's the purpose of this parameter? 
Does this means that by default it doesn't handle detected action? 

I am wondering if it would make sense to have this parameter in the matching rule instead 
https://github.com/updatecli/updatecli/blob/main/pkg/plugins/autodiscovery/githubaction/matchingRule.go

So we could use "only/ignore"

Reply:
I've followed what was done for the workflowfiles, but happy to do a more thorough pass